src/edu/washington/escience/myria/operator/Operator.java

```diff
@@ -107,6 +108,22 @@ public ProfilingLogger getProfilingLogger() {
   }
 
   /**
+   *
+   * @return PythonFunctionRegistrar for operator.
+   * @throws DbException  in case of error.
+   */
+  public PythonFunctionRegistrar getPythonFunctionRegistrar() throws DbException {
```

Comment:
My understanding is, `pyFuncRegistrar` is only used by `PythonUDFEvaluator`, so you don't need to add this function to `Operator`, and many aggregator-related methods. for example, `AggUtils.allocateAggs()`. Just get `pyFuncRegistrar` from the worker in `PythonUDFEvaluator`.